### DISCUSSION TITLE
Backup Of main nodes

### DISCUSSION THREAD

**Discussion by cerberus10748** [Security-Onion-Solutions/securityonion#6032] [Unsupported Versions] [answered]
Hello, 
I  am looking for a way to back up my main nodes for security onion. Such as my manager. I am not worried about my foward sensors since they are easy to restore, but I am worried about losing my manger and having to restore it. Can i back up the whole machine to an external drive via a command? specific files to load if it fails. Thank you in advance.

**Comment by dougburks** [comment, marked as answer]
Please see the documentation:
https://docs.securityonion.net/en/2.3/backups.html
